### PR TITLE
replace same by deepEqual

### PR DESCRIPTION
**Pull request by denar90**
PR for issue #1336 
Running `npm test` I got `✖ 2 problems (1 error, 1 warning)` but looking at [build](https://travis-ci.org/brunch/brunch/jobs/127817268) I think this problem was present before, so I think, I didn't break  anything.

### CODE REVIEW

**Comment by denar90** [comment]
Added some test fixes, but I feel that tomorrow I'll get more elegant solution.

File: lib/config.js [hunk 1]
```diff
@@ -631,6 +631,7 @@ const tryToLoad = (configPath, fallbackHandler) => {
   try {
     pkg = require(sysPath.resolve('.', 'package.json'));
   } catch (e) {
+    logger.error(e);
```
Comment (goshakkk) [inline lib/config.js]: We don't actually need this logged. We are loading package.json here simply to try and recover from brunch-config require errors

Comment (denar90) [inline lib/config.js]: But tests are crashing with empty `catch`

Comment (goshakkk) [inline lib/config.js]: Just adding a comment or something in there should have made eslint happy

like

```
} catch (e) {
  // no-op
}
```

Comment (denar90) [inline lib/config.js]: go it

**Pull request merged**